vector/v.what.rast3/testsuite/test.v.what.rast3.sh

```diff
@@ -11,14 +11,28 @@ r3.mapcalc --o expr="plume = double(col() + row() + depth())"
 # v.random --o -z seed=1 output=random_points npoints=10 zmin=0  zmax=50
 # v.out.ascii --o format=standard input=random_points output=random_points.txt
 
-v.in.ascii --o -z format=standard input=random_points.txt output=random_points
+v.in.ascii --o -z format=standard input=data/random_points.ref output=random_points
+v.out.ascii --o format=standard input=random_points > data/random_points.txt
```

Comment:
I agree it looks like that. Initially, there was some ".ref" and ".txt" present, but the ".txt" files got changed with trailing spaces removed and EOF changed. So the tests failed at first since the ref and output won't match. So, to make sure that it stays working, and since I didn't know if the import/export was properly tested, I went on the golden-copy approach and verifying that the inputs, the preconditions of the tests, are initially valid before going forward. It's motivated by the fact that the info was present, but doing nothing.

Now, with my WIP of code coverage, I would be able to check if I remove the line if the coverage stays the same. I'd also check that running tests through shell scripts actually show up in code coverage (I'll try to find a not touched module and do something simple in a shell script and look if I can see the change in coverage).

Reply:
I now have my answer: it doesn't change the coverage at all, even when limited to the scripts folder. So I'll remove that extra line